net/AvatarMaskView.h

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
Its just for learning. I implemented two variants, and chose one of them. In real project I wount looking for more then one solution